include/sark.h

```diff
@@ -167,6 +167,7 @@ enum sark_aplx_command {
 
 #define ALLOC_LOCK      1       //!< Lock this operation
 #define ALLOC_ID        2       //!< Use supplied AppID
+#define ALLOC_TAG_RETRY 4       //!< If tag in use, just return address
```

Comment:
Probably should also say that that's only true if the tagged allocation _also_ has the same size.

Reply:
I have updated the docs as requested.  I am not certain this is in the right form for doxygen - tell me if not!